WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -108,6 +107,31 @@ def show_user_by_wca_id
     show_user(user)
   end
 
+  def show_registrations_by_comp_id
```

Comment:
I feel like this would make more sense in our existing [`Api::V0::CompetitionsController`](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/controllers/api/v0/competitions_controller.rb).